eodag/api/core.py

```diff
@@ -2092,7 +2092,8 @@ def list_queryables(
         self,
         provider: Optional[str] = None,
         product_type: Optional[str] = None,
-    ) -> Dict[str, Tuple[Annotated[Any, FieldInfo], Any]]:
+        **kwargs: Optional[Dict[str, Any]],
+    ) -> Dict[str, Tuple[Annotated, Any]]:
```

Comment:
- return type is still of type Dict[str, Tuple[Annotated[Any, FieldInfo], Any]]
- **kwargs should be of type Any
- **kwargs missing in docstring